src/main/java/sg/ncl/MainController.java

```diff
@@ -3494,6 +3486,514 @@ public String adminEnergy(Model model,
         return "energy_usage";
     }
 
+    private ProjectDetails getProjectDetails(JSONObject jsonObject) {
+        ProjectDetails projectDetails = new ProjectDetails();
+        projectDetails.setId(jsonObject.getInt("id"));
+        projectDetails.setOrganisationType(jsonObject.getString("organisationType"));
+        projectDetails.setOrganisationName(jsonObject.getString("organisationName"));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "organisationName" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)